edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -618,23 +626,44 @@ impl ModuleRuntime for DockerModuleRuntime {
             return Box::new(future::err(Error::from(err)));
         }
 
+        let (signal_socket_created, receiver): (Sender<()>, Receiver<()>) = oneshot::channel();
```

Comment:
Name the sender and receiver consistently.

Reply:
Fixed